api/src/api/schemas/response_schema.py

```diff
@@ -58,3 +58,11 @@ class ErrorResponseSchema(Schema):
             "example": "550e8400-e29b-41d4-a716-446655440000",
         }
     )
+
+
+class FileResponseSchema(Schema):
+    download_path = fields.String(
+        metadata={
+            "description": "The file's download path",
+        },
+    )
```

Comment:
I don't have a strong opinion on whether we want to do this, but if we're going to setup a base class, we should copy all the common fields, and not need to define them in the two derived classes (download_path, file_size_bytes, created_at, updated_at).

At

Reply:
I think it's a good idea as we may use it in the future and it would be duplicated otherwise. I adapted these changes for the common fields and left `OpportunityAttachmentV1Schema` with the fields not relevant to the `ExtractMetadataResponseSchema`